deploy/7_AddBassetInBasketManager.ts

```diff
@@ -0,0 +1,44 @@
+import { constants } from "ethers";
+import { DeployFunction } from "hardhat-deploy/types";
+
+const func: DeployFunction = async ({
+  deployments,
+  getNamedAccounts,
+}) => {
+  const network = deployments.getNetworkName();
+  const {deployer} = await getNamedAccounts();
+  let bAssets = [
```

Comment:
pls set bAssets by network in one if...else block.

Reply:
updated in this [commit](https://github.com/DistributedCollective/mynt/pull/5/commits/b791ecb597f6586e57a2ee71466f6b0984cb8889)